tasks/parse-pdr/index.js

```diff
@@ -170,12 +174,14 @@ const extractGranuleId = (fileName, regex) => {
  * @param {Object} params.fileGroup - PDR FILE_GROUP object
  * @param {string} params.pdrName - name of the PDR for error reporting
  * @param {Object} params.collectionConfigStore - collectionConfigStore
- * @returns {Object} granule object
+ * @param {boolean} params.uniquifyGranuleId
```

Comment:
Reviewers: looking for feedback on this jsdoc - I should update this docstring, but in doing so I wind up needing to define types for `packages/pvl` which might spiral in scope.   I'd like a second opinion on scope.